src/tmpfiles/tmpfiles.c

```diff
@@ -3724,6 +3740,14 @@ static int parse_line(
                                 CREATE_SUBVOLUME_INHERIT_QUOTA,
                                 CREATE_SUBVOLUME_NEW_QUOTA) ? 0755 : 0644;
 
+        if (missing_user_or_group && (i.mode & ~0777)) {
+                /* Refuse any special bits for nodes where we couldn't resolve the ownership properly. */
+                mode_t new = i.mode & 0777;
+                log_syntax(NULL, LOG_ERR, fname, line, r,
```

Comment:
the fifth arg should be 0 i guess.

and LOG_ERR should be LOG_NOTICE, after all we don't actually fail anything here.